app/facades/food_facade.rb

```diff
@@ -0,0 +1,6 @@
+class FoodFacade
+  def self.search(query)
+    json = FoodDataService.search(query)
+    json[:foods].map { |food| Food.new(food) }
```

Comment:
This works. This is also where you could have grabbed the totalHits. Then you could save that to a variable and pass both the food objects and the totalHits value back to the controller as either a Hash or an Array that you could deconstruct into separate instance variables. This would allow you to only hit the external API once. Faster code!